utils.go

```diff
@@ -46,6 +46,12 @@ func readFromStdIn() string {
 }
 
 func promptForYes(message string) bool {
+
+	// Don't prompt for confirmation if the quiet flag is enabled
+	if *oQuiet {
```

Comment:
I'm not really sure if it makes sense to refer to a flag from the utils file? Though it seems the most consistent place to just skip all confirmation prompts from here.

Reply:
Maybe pass along the flag to the function, if that feels less nasty?

Reply:
Not sure, that feels more messy to me because then every caller needs to know where to get the setting from.
I'm mostly just not sure if it's bad practice or not to use these kinds of variables from a utility function

Reply:
the flags are global on purpose, to keep it simple. Other tools such as gmig use a cli pkg which provides the means to pass flags and have operation local scope.

Maybe someday we will rewrite kiya to use that pkg